### ISSUE TITLE
Multiple population names

### ISSUE THREAD

**Issue by TCLamnidis** [stschiff/sequenceTools#34]
A small quality of life suggestion: allowing users to specify the population of each sample from the CLI.

Currently, users can either get the default Population label (`Unknown`), or specify a single population label for all the individuals. 
It would be nice addition if users could provide a comma separated list of population labels (same length and order as `--sampleNames`), to be used as the population of the respective samples. 

E.g.:
```
pileupCaller \
    -f 1240K_covered_in_JK2067_downsampled_s0.1.numeric_chromosomes.snp \
    --randomHaploid  -e my_dataset \
    --sampleNames Sample_1,Sample2,Sample3/
    --samplePopName Pop1,Pop2,Pop3
 ```
 
 To get an output ind like:
 ```
 Sample1    U    Pop1
 Sample2    U    Pop2
 Sample3    U    Pop3
 ```

**Comment by stschiff** [comment]
I agree. Put on my Todo list!

**Comment by stschiff** [comment]
fixed in v1.5.4.0
